File: src/sandbox/task_config.py
from dataclasses import dataclass

@dataclass
class TaskConfig:
    # ЗАМЕНА: grid_width, grid_height → scene_width, scene_height (float)
    scene_width: float = 1000.0    # Пиксели
    scene_height: float = 1000.0   # Пиксели
    # УДАЛЕНО: tolerance_px, min_trace_ratio, level - не используются в Python

File: src/Learn2Code.Core/Models/TaskConfig.cs
using Learn2Code.Core.Enums;

namespace Learn2Code.Core.Models;

public class TaskConfig
{
    public double SceneWidth { get; set; } = 1000.0;
    public double SceneHeight { get; set; } = 1000.0;

    public double TolerancePx { get; set; } = 5.0;
    public double MinTraceRatio { get; set; } = 0.7;
    public CheckLevel Level { get; set; } = CheckLevel.Normal;
}